Addressing Paul's DISCUSS (#38)
ietf-rats-wg/draft-ietf-rats-uccs: draft-ietf-rats-uccs.md

protocol identified by ALPN) MUST specify the roles of the endpoints
in a fashion that the security properties of conveying UCCS via a
Secure Channel between the roles are well-defined.

> The Secure Channel cryptographic strength should be at least as strong as the cryptographic keys it is transporting through this Secure Channel.

Reviewer: avoid some prose redundancy
```suggestion
The cryptographic strength of the Secure Channel must be at least as strong as the cryptographic keys it transports.
```

Author: poetry

Reviewer 2: Why did the should change to a must?  That is wrong.
(Why must the strength be strong? That's just clumsy.)
